e2e/cypress/tests/integration/boards/create_board_spec.ts

```diff
@@ -47,6 +47,54 @@ describe('Create and delete board / card', () => {
             should('have.value', 'Testing');
     });
 
+    it('MM-T4275 Set up Board description', () => {
+        cy.visit('/boards');
+
+        // # Create empty board and change tile to Testing
```

Comment:
```suggestion
        // # Create an empty board and change tile to Testing
```

Reply:
Thanks for the approval. I have made the changes suggested by you.